### PR TITLE
May heartbeat draft

### PR DESCRIPTION
**Pull request by andronovhopf**
Working on a header picture currently

### CODE REVIEW

File: content/blog/2020-05-14-may-20-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+title: May '20 DVC❤️Heartbeat
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-heartbeat.md:2]: URL should be `may-20-dvc-heartbeat.md`?

File: content/blog/2020-05-14-may-20-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+picture: 2020-05-14/May_20_Heartbeat.png
+pictureComment: A big hello from DVC mascot DeeVee.
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/april-20-heartbeat/347
+tags:
+  - Heartbeat
+  - Plots
+  - MLOps
+  - Meetup
+  - Season of Docs
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-heartbeat.md:25]: `Google Season of Docs`- https://dvc.org/blog/tags/google-season-of-docs

File: content/blog/2020-05-14-may-20-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  - Ambassador
+---
+
+Welcome to the May Heartbeat, our
+[monthly roundup of cool happenings](https://dvc.org/blog/tags/heartbeat), good
+reads and other bright spots in our community.
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-heartbeat.md:31]: I think it would be helpful to describe the content a little bit wih some links to sections inside

Comment (shcheklein) [reply content/blog/2020-05-14-may-20-heartbeat.md:31]: here or in the long description

File: content/blog/2020-05-14-may-20-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+https://twitter.com/braaannigan/status/1257918525345234949
+
+https://twitter.com/josh_wills/status/1249774857614553097
+
+https://twitter.com/tcgarvin/status/1258855168436813826
+
+_Thank you, thank you very much._
+
+<iframe src="https://giphy.com/embed/gJ2sDSKAQHUCIYUFhx" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/TheWiggles-wiggles-the-thewiggles-gJ2sDSKAQHUCIYUFhx">via GIPHY</a></p>
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-heartbeat.md:175]: have you tried to embed it the same way we do Twitter links?

File: content/blog/2020-05-14-may-20-heartbeat.md (hunk 1 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-heartbeat.md:2]: fixed

File: content/blog/2020-05-14-may-20-heartbeat.md (hunk 2 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-heartbeat.md:25]: fixed

File: content/blog/2020-05-14-may-20-heartbeat.md (hunk 4 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-heartbeat.md:175]: that worked!

File: content/blog/2020-05-14-may-20-heartbeat.md (hunk 3 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-heartbeat.md:31]: i added some details in the description and some links in the intro

File: content/blog/2020-05-14-may-20-dvc-heartbeat.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title: May '20 DVC❤️Heartbeat
 date: 2020-05-14
 description: |
   Catch up on new DVC releases, talks, and projects in our community. 
-  This month, learn what we're up to in MLOps, CI/CD, and the 
+  This month, learn about new features in the momentous DVC 1.0 release, virtual ways to get involved, and more from the
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-dvc-heartbeat.md:6]: pleas, make it <80 symbols length

Comment (shcheklein) [reply content/blog/2020-05-14-may-20-dvc-heartbeat.md:6]: I mean, wrap it at ~80

File: content/blog/2020-05-14-may-20-dvc-heartbeat.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+https://twitter.com/braaannigan/status/1257918525345234949
+
+https://twitter.com/josh_wills/status/1249774857614553097
+
+https://twitter.com/tcgarvin/status/1258855168436813826
+
+_Thank you, thank you very much._
+
+<iframe src="https://giphy.com/embed/gJ2sDSKAQHUCIYUFhx" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/TheWiggles-wiggles-the-thewiggles-gJ2sDSKAQHUCIYUFhx">via GIPHY</a></p>
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-dvc-heartbeat.md:175]: so, are we changing this? (probably, didn't get your answer - was it a confirmation or not :) )

File: content/blog/2020-05-14-may-20-dvc-heartbeat.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+  Every month we share news, findings, interesting reads, community takeaways,
+  and everything else along the way.
+
+  Look here for updates about [DVC](https://dvc.org), our journey as a startup,
+  projects by our users and big ideas about best practices in ML and data
+  science.
+picture: 2020-05-14/May_20_Heartbeat.png
+pictureComment: A big hello from DVC mascot DeeVee.
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/april-20-heartbeat/347
```
Comment (shcheklein) [inline content/blog/2020-05-14-may-20-dvc-heartbeat.md:19]: update URL?

File: content/blog/2020-05-14-may-20-dvc-heartbeat.md (hunk 6 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-dvc-heartbeat.md:175]: ah yes, missed this one! will be changing it to just the link, which embeds like a twitter link

File: content/blog/2020-05-14-may-20-dvc-heartbeat.md (hunk 5 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-dvc-heartbeat.md]: should be fixed now!

File: content/blog/2020-05-14-may-20-dvc-heartbeat.md (hunk 7 above)
Comment (andronovhopf) [reply content/blog/2020-05-14-may-20-dvc-heartbeat.md]: done

**Pull request merged**